ios/escape/escape/Models/GroupModel.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  GroupModel.swift
+//  escape
+//
+//  Created by Claude on 2025/11/02.
+//
+
+import Foundation
+
+// MARK: - Group Model
+
+/// Represents a group from the database
+struct Group: Codable, Identifiable, Equatable {
```

Comment:
How about rename this to something elese. As you have noticed, it overlapped with the SwiftUI component